src/lib/cloudfiles.js

```diff
@@ -31,6 +31,8 @@ async function getCloudFile(file) {
     }
 }
 
+getCloudFile.CACHE_DIR = path.join(homedir, ".nui", "cache");
```

Comment:
do we want to change this to `.assetcompute`?

Reply:
This is the next step with [NUI-556](https://jira.corp.adobe.com/browse/NUI-556). This PR just adds initial unit tests and not meant to change behavior.